lib/options_validator.js

```diff
@@ -27,14 +27,20 @@ function createValidationFunction(optionsSchema) {
  * @param {object} optionsSchema A schema specifying possible options and their types, defaults,
  *   deprecation status, and whether or not they are required.
  * @param {object} providedOptions The options passed into the operation.
- * @param {object} validationOptions Options for the validation itself.
+ * @param {object} overrideOptions Optional. The values for options that need to check inheritance, like readPreference.
```

Comment:
these are signaled as optional by naming them with brackets around the option name, e.g. 
```
@param {object} [overrideOptions] ...
```